### ISSUE TITLE
ACL/Roles container does not meet WCAG standards for accesible text

### ISSUE THREAD

**Issue by Raruto** [g3w-suite/g3w-admin#470]
### Checklist

- [x] I've searched through the existing [issues](https://github.com/g3w-suite/g3w-admin/issues) and this bug has never been reported before

### Subject of the issue

As you can see from the attached images, the text inside the **ACL/Roles** container can be difficult to read:

![image](https://user-images.githubusercontent.com/9614886/223059570-eaf92a3c-69e8-4893-bb7d-4fe6e4c00701.png)

![image](https://user-images.githubusercontent.com/9614886/223060207-5a65073d-fb41-47e6-bf80-7284ab88f135.png)

### Steps to reproduce

1. login as admin user: http://localhost:8000/en/admin/
2. go to `Dashboard > Users > User update`: eg. http://localhost:8000/en/admin/users/update/2/
3. open Developer Tools (`ctrl+shift+i`) and navigate to the Accessibility tab
4. check the [perceivable color contrast](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast) value between text and background color of the **ACL/Roles** container

### Environment

- g3w-admin: __3.5.dev-20230224161958__
- browser: __Firefox 110__
- operating system: __Windows 10 (64 bit)__

### Link to your project

http://localhost:8000/en/admin/users/update/2/
